.talismanrc

```diff
@@ -1,349 +1,3 @@
-    - filename: app/celery/letters_pdf_tasks.py
-      checksum: 17817004fe9477d2a748f4a9a6d63e87c25e60c92f3ccf2362bf6f32d542f1e1
-    - filename: app/notifications/rest.py
-      checksum: d22b6350a68e06c1c71893ca41b2217eb8cd61a21408baba79ca77b0bc0a37db
-    - filename: .github/workflows/dev_deploy.yml
-      checksum: 217b1dbca27958380523fce8c4e3c56af5e73e3ae914a2117a2574739f7d15a7
-    - filename: tests/env_vars
-      checksum: 2a33fe8099baf5449abd0b4deefb7796e770ad504734cf12c21a2f1e5f357a3d
-    - filename: lambda_functions/two_way_sms/two_way_sms_v2.py
-      checksum: bb0eb348130e9fa435d835c5c92548b21cf93ef2d4ab909e92b28c5bd7cdf91a
-    - filename: lambda_functions/nightly_billing_bigquery_upload/nightly_billing_stats_upload_lambda.py
-      checksum: 3cef32b214b5aa8d21c47493049aed0a9cbf8340fe6dab2ffa8461f7216fce8b
-    - filename: app/dao/fact_billing_dao.py
-      checksum: ee78cb3f234ca6b2fb28a2941ec4a44104d6866b8bb274c195847b8f609a8c88
-    - filename: app/user/rest.py
-      checksum: d04d6378dd6ce8b0046d6dc2c106ca87cf93ee555e03821e7970570c4b132450
-    - filename: tests/app/service/test_api_key_endpoints.py
-      checksum: 1ac2f66bdf3a93a8c0c71ea97a3f464fc885f98034012b5c85de05d883b324c0
-    - filename: certs/DigiCert-Global-Root-G2.cer
-      checksum: ae4a406af9f5ccd0c8b7d2439076327fb131760d29043f8488a22f6eff26a27e
-    - filename: certs/DigiCert-Global-G2-TLS-RSA-SHA256-2020-CA1.cer
-      checksum: 120ea4457eac8216580853c44751a329b74cc29fafc9064105ee73d9a9c2bdc8
-    - filename: migrations/versions/0359_communication_items_unique.py
-      checksum: 1c963f08a88caf7de46e27f89fdc7af37b38a61bb62b10498bfca26bc29d792e
-    - filename: app/schema_validation/__init__.py
-      checksum: a2ee4584781782e5154ae0641f85777bc263d6f1e6f9fe1cfd370fff0a869418
-    - filename: app/v3/notifications/rest.py
-      checksum: 7b12a5909efcd0e34c97661bee128cebd2968c186ce455699fcc668a4470244f
-    - filename: app/celery/v3/notification_tasks.py
-      checksum: 68a0f794fd010f7c2916de4e1af28aabba4e6465a8020f8f3b9f5f2e4d1bfa1a
-    - filename: tests/app/v3/notifications/test_rest.py
-      checksum: f970efab73ad52e3e203fe0805925929033a72d9975ec1deb077331e9409f06f
-    - filename: app/celery/scheduled_tasks.py
-      checksum: f435f4ea9bd8d5484b01ad10936b5ef22b1b502377065e0de0288f0b9efd1d7f
-    - filename: app/notifications/send_notifications.py
-      checksum: c91dd74d45c6e4ea76e66964688bab6b6d65275857d9422061ff7e8b663748d8
-    - filename: tests/app/notifications/test_send_notifications.py
-      checksum: 23f7e8057a8fe922a3115c79c1898c9238971a8a24c9af5b40baf0a334200925
-    - filename: tests/app/celery/test_scheduled_tasks.py
-      checksum: 2893fb34960e6a1498a8da529d99c668806e6a156996b2914f807f768b5148bb
-    - filename: tests/lambda_functions/two_way_sms/test_two_way_sms_v2.py
-      checksum: 468b6d1fe14a7a621a3807ebcaee1d4701e57004e99a0f8c386731dc14df2970
+- filename: app/celery/letters_pdf_tasks.py
```

Comment:
This looks good to me. Quickly looking through there are some changes I personally wouldn't have made because I think it makes some things a little less readable with it all being on one line, but not a huge deal. 

Just one question. It looks like this file was reset. Why is that?

Reply:
Solid question! It was growing without end. For example, the first file does not even exist. 

Ideally, these would be _permanent exemptions_ for things that were not actually security threats and the exemptions would automatically update the checksum if we okay'd it. Deleted files should trigger cleanup automatically too.

What benefit is there to having this file? We have to paste the new values in every time the file changes. The tool is not smart enough to realize the exempted part did not change (since it only stores checksum), nor update an existing checksum if we okay'd it. 

Talisman's only usefulness I see is in identifying potential security violations before they happen. This file is of zero use to us I believe. Maybe [newer versions of Talisman](https://github.com/thoughtworks/talisman/releases) are more useful?

If I'm off-base here please help me understand where I took a wrong turn.

Reply:
+1 I brought this up last year. Adding it to the Pain Points table.

Reply:
Those are valid points and you're right. Talisman isn't a very smart tool, it just exists to help prevent secrets from being leaked, as you said. 

I'm on board with this file being cleaned up, just wanted to know the reasons behind it. Thanks!